pages/what-we-do/overview.md

```diff
@@ -21,28 +21,37 @@ tags:
 </div>
 
 <div class="content-block">
-  <div class="container container--sm vertical-padding">
+  <div class="container container--xs vertical-padding">
 
 The National Leadership Centre was launched by the Chancellor in 2018 as a three year pilot, following the recommendations of [the Public Services Leadership Taskforce](https://www.gov.uk/government/publications/national-leadership-centre). The Taskforce was established in 2017 to advise on the scope for a new body to support public service leaders.
 
-<div class="header-highlight">
+<div class="pdf-left-block">
+	<div class="pdf-left-block__pdf">
+		<a href="https://assets.publishing.service.gov.uk/government/uploads/system/uploads/attachment_data/file/799567/6.4846_CO_CPSL-Report_A4-P_WEB_NoLogo__002.pdf">
+		<img src="../../static/img/taskforce-report-pdf.png" alt="Image of taskforce report"/>
+		<span>October 2018: Better Public Services Report by the Public Services Leadership Taskforce</span>
+		</a>
+		<span>PDF, 1010KB, 52 pages</span>
+	</div>
+	<div class="header-highlight">
 
 ### The Taskforce found that the scale and complexity of demands on our public services continue to grow. Modern society is becoming more diverse in its needs; technology is evolving at an accelerating rate; and the UK is striving for greater productivity within an increasingly complex world.
 
+   </div>
 </div>
 
 Although the Taskforce was clear that public service leadership is not a silver bullet for these challenges, they found that the weight of evidence suggests that bolstering public service leaders does make a difference. The Taskforce also found that many leaders are struggling with the strategic complexity of the services they lead, while experiencing a sense of isolation amid the challenges that come with becoming the most senior person of a public service organisation.
 
 The Taskforce made [a number of recommendations](https://www.gov.uk/government/publications/national-leadership-centre) in their report and, in response, the government created the National Leadership Centre to support leaders of public services in England to improve the services they run.
 
-### **Related Links:**
```

Comment:
I thin kwe can remove this link but we probably should keep the email address for the accessible format

Reply:
No problem - kept the accessibility link at the bottom 👍